### PR TITLE
Generating models

### PR DESCRIPTION
**Pull request by djimenez017**
MAde the changes requested from the previous commit

### CODE REVIEW

File: app/models/saved_yardsale.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+class SavedYardsale < ApplicationRecord
+  belongs_to :user
```
Comment (kervin5) [inline app/models/saved_yardsale.rb]: I think this also belong to yard_sale.
You also need to specify the name of the table that was created with the migration. Was it user_yardsales? I can't remember

File: app/models/yard_sale.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+class YardSale < ApplicationRecord
+  belongs_to :user
+  has_many :images
+  has_one :address
+  has_many :yardsale_tags
```
Comment (kervin5) [inline app/models/yard_sale.rb]: We don't need this is we're using act as taggable. Instead you can use acts_as_taggable_on :tags

File: app/models/user.rb [hunk 3]
```diff
@@ -0,0 +1,17 @@
+class User < ApplicationRecord
+  has_many :yard_sales
+  has_one :saved_yardsale
+
```
Comment (kervin5) [inline app/models/user.rb:5]: Se need to specify that the user is a tagger with act_as_tagger

**Review by kervin5 — CHANGES_REQUESTED** [review]
I think it's looking great. I left some additional comments and I'm requesting changes because the migration hasn't been run. Running it helps to know if there are any possible logic issues.

File: app/models/user.rb [hunk 4]
```diff
@@ -0,0 +1,17 @@
+class User < ApplicationRecord
+  has_many :yard_sales
+  has_one :saved_yardsale
```
Comment (kervin5) [inline app/models/user.rb]: Shouldn't this be has_many? Because a user can save many yard sales?

**Review by kervin5 — CHANGES_REQUESTED** [review]

File: app/models/user.rb (hunk 3 above)
Comment (djimenez017) [reply app/models/user.rb:5]: has been updated

File: app/models/user.rb (hunk 4 above)
Comment (djimenez017) [reply app/models/user.rb]: has been updated

File: app/models/user.rb [hunk 5]
```diff
@@ -0,0 +1,18 @@
+class User < ApplicationRecord
+  has_many :yard_sales
+  has_many :saved_yardsale
```
Comment (kervin5) [inline app/models/user.rb]: This should be saved_yardsales. Plural.

**Review by kervin5 — CHANGES_REQUESTED** [review]
One small comment. Almost there.

File: app/models/saved_yardsale.rb (hunk 1 above)
Comment (kervin5) [reply app/models/saved_yardsale.rb]: I think we still need to specify the name of the table "users_yard_sales" manually for this model. https://stackoverflow.com/questions/4613574/how-do-i-explicitly-specify-a-models-table-name-mapping-in-rails

**Review by kervin5 — CHANGES_REQUESTED** [review]
Just one small comment about the saved yard sales model.

**Review by kervin5 — APPROVED** [review]
🎉

**Pull request merged**